samples/logging/hostcustom/Version_6/Sample/app.config

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="NServiceBus.Core" publicKeyToken="9fc386479f8a226c" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-6.0.0.0" newVersion="6.0.0.0" />
```

Comment:
log4net versions down to the patch version. so as soon as log4net release a new version after we have release nsb.log4net people need to have a binding redirect. hence it exists in the old samples. possibly it is not required currently since nsb.log4net was updated recently, but will have to re-add it as soon as log4net is released again. 

strong naming: broken by design